modal-go/cls.go

```diff
@@ -51,7 +51,7 @@ func ClsLookup(ctx context.Context, appName string, name string, options LookupO
 	parameterInfo := serviceFunction.GetHandleMetadata().GetClassParameterInfo()
 	schema := parameterInfo.GetSchema()
 	if len(schema) > 0 && parameterInfo.GetFormat() != pb.ClassParameterInfo_PARAM_SERIALIZATION_FORMAT_PROTO {
-		return nil, fmt.Errorf("unsupported parameter format: %v", parameterInfo.GetFormat())
+		return nil, fmt.Errorf("unsupported parameter format: %w", parameterInfo.GetFormat())
```

Comment:
Hm this needs to be `%v` since it's not an error, so `%w` would break. https://pkg.go.dev/golang.org/x/xerrors#Errorf

> It is invalid to include more than one %w verb or to supply it with an operand that does not implement the error interface. The %w verb is otherwise a synonym for %v.